_posts/2022-08-08-Interview_Joshua.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Interview Joshua Blomgren"
+author: Chongye Feng
+---
+
+# Joshua Blomgren
+
+## From:
+
+San Jose, CA
+
+It is in the south bay area, the silicon valley. It is hot and does not snow. 
+
+## Living in:
+
+East Bay area. Joshua is living in the city called Doublim.
```

Comment:
I found the city in the map. It is where the outlet located in.

Reply:
Yeah hahaha you are right!